f2/src/LandingPage.js

```diff
@@ -1,24 +1,27 @@
 /* eslint-disable react/no-unescaped-entities */
 import React from 'react'
+import { useLingui } from '@lingui/react'
 import { Route } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import { Trans } from '@lingui/macro'
 import { P } from './components/paragraph'
 import { Button } from './components/button'
-import { H1 } from './components/header'
+import { H1, H3 } from './components/header'
```

Comment:
The `H3`s should be `H2`s for a11y (ie to keep the H1 -> H2 -> H3 -> etc hierarchy). We can style them to be the same size as `H2`s.